template/gcc.tex

```diff
@@ -82,7 +82,7 @@
 \rhead{GCC! $date$}
 \lhead{$title$}
 \cfoot{\thepage}
-\fancyfoot[L]{\includegraphics[width=2cm]{../logo_gcc_court.pdf}}
+\fancyfoot[L]{\includegraphics[width=1.7cm]{../logo_gcc_court.pdf}}
```

Comment:
Use [width=XX\linewidth] or \textwidth with XX a number from 0 to 1 which is the percentage of the command. So for example: 0.3\linewidth == 30% of line width

let me know if it is not clear